src/Generators/Microsoft.Gen.Logging/Parsing/Resources.resx

```diff
@@ -309,4 +303,28 @@
   <data name="TemplateStartsWithAtSymbolTitle" xml:space="preserve">
     <value>A logging message template starts with "@"</value>
   </data>
+  <data name="CantUseDataClassificationWithLogPropertiesOrTagProviderMessage" xml:space="preserve">
+    <value>Cant combine the [LogProperties] or [TagProvider] attributes with data classification attributes</value>
+  </data>
+  <data name="CantUseDataClassificationWithLogPropertiesOrTagProviderTitle" xml:space="preserve">
+    <value>Cant combine the [LogProperties] or [TagProvider] attributes with data classification attributes</value>
+  </data>
+  <data name="CantMixAttributesMessage" xml:space="preserve">
+    <value>Can only have one of [LogProperties], [TagProvider], and [LogPropertyIgnore]</value>
```

Comment:
Does it make sense to have `LogPropertyIgnore` since we don't do transitive traversal by default?

Reply:
For a given property, you can ask:

* Output the property's string (no attribute)
* Ignore this property and don't output it (LogPropertyIgnore)
* Output the properties of the property's object (LogProperties)
* Invoke the tag collector to get properties for the property's object (TagProvider)

So I think they're all mutually-exclusive.

Reply:
Ah, I see